scripts/kernel-version-check.py

```diff
@@ -3,12 +3,17 @@
 # zenoss-inspector-info
 # zenoss-inspector-deps uname-a.sh
 import re
+import platform
 
 def main():
-    with open("uname-a.sh.stdout", 'r') as f:
-        line = f.read()
-        if tuple(map(int, re.search(r'^[^ ]+ [^ ]+ (\d+)\.(\d+)\.(\d+)-(\d+)', line).groups())) < (3,10,0,327):
-            print 'Kernel version must be 3.10.0-327 or higher, please update your kernel'
+    # Check for Centos/RHEL 7.2
+    if tuple(map(int, platform.linux_distribution()[1].split('.'))) >= (7,2):
+        with open("uname-a.sh.stdout", 'r') as f:
+            line = f.read()
+            kver = tuple(map(int, re.search(r'^[^ ]+ [^ ]+ (\d+)\.(\d+)\.(\d+)-(\d+)', line).groups())) 
+            if kver < (3,10,0,327):
+                if kver > (3,10,0,229):
```

Comment:
I think that comment might be more descriptive and say why we are checking for those kernel versions.

Also those 2 `if`s could be replaced by a single one if it matters to you
```python
if (3,10,0,229) < kver < (3,10,0,327):
```